src/store/actions/resource.js

```diff
@@ -449,7 +449,10 @@ export const removeShareActivity = async (activityId, resourceName) => {
   }
 };
 
-export const loadH5pShareResource = (activityId) => resourceService.loadH5pShared(activityId);
+export const loadH5pShareResource = async (activityId) => {
+  const result = await resourceService.loadH5pShared(activityId);
+  return result;
+};
```

Comment:
Do we need this change?
I already saw this several times but I don't think it is necessary